package.json

```diff
@@ -9,29 +9,31 @@
   "license": "MIT",
   "description": "Teleport frontend template to integrate a bundled react app",
   "dependencies": {
-    "react": "15.0.2",
-		"react-dom": "15.0.2"
+    "react": "^15.5.4",
+		"react-dom": "^15.5.4",
   },
   "devDependencies": {
-    "autoprefixer": "^6.3.1",
+    "autoprefixer": "^7.1.1",
     "babel-core": "^6.21.0",
     "babel-loader": "^6.2.10",
+    "babel-polyfill": "^6.23.0",
     "babel-preset-latest": "^6.22.0",
 		"babel-preset-react": "^6.22.0",
     "css-loader": "^0.23.1",
+    "extract-text-webpack-plugin": "^2.1.0",
     "gzip-size": "^3.0.0",
 		"node-sass": "^3.7.0",
 		"postcss-loader": "0.9.1",
     "react-hot-loader": "^3.0.0-beta.1",
     "sass-loader": "^3.1.2",
 		"style-loader": "^0.13.0",
-    "webpack": "^1.13.0",
-    "webpack-dev-server": "^1.14.1"
+    "webpack": "^2.6.1",
+		"webpack-dev-server": "^2.4.5"
```

Comment:
OK

I just wondering why you also add the webpack-dev-middleware ?
Because either you use the webpack dev server (and use directly the headers option), or you use the webpack dev middleware inside your predefined express server ?